chess_env.py

```diff
@@ -38,6 +42,7 @@ def get_state(self):
         Get the current state of the environment.
         :return:
         """
+        return self
```

Comment:
I think that returning a FEN string would be best here. We'll have to save a lot of sates in memory when doing the tree search so we want the state to be as compressed as possible